virt/monitoring/virt-runbooks.adoc

```diff
@@ -299,8 +324,9 @@ Runbooks for the {VirtProductName} Operator are maintained in the link:https://g
 [id="virt-runbook-virtualmachinecrcerrors_{context}"]
 == VirtualMachineCRCErrors
 
-* The runbook for the `VirtualMachineCRCErrors` alert is deprecated because the alert was renamed to `VMStorageClassWarning`. 
-** link:https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/VMStorageClassWarning.md[View the runbook] for the `VMStorageClassWarning` alert.
+* The `VirtualMachineCRCErrors` alert is link:https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/VirtualMachineCRCErrors.md[deprecated].
+
+The alert is now called `VMStorageClassWarning`. See below.
```

Comment:
```suggestion
The alert is now called `VMStorageClassWarning`.
```
According to the [Accessibility](https://docs.google.com/spreadsheets/d/1mLU9oxRWFUANbBbpRnfxL0-eIh5EVtYGfd_kgYZSeSE/edit?gid=836060893#gid=836060893) section of the SSG: "Avoid indicating direction, such as left, right, above and below, since these have no meaning to a screen reader."